torch/lib/c10d/ProcessGroupNCCL.hpp

```diff
@@ -155,11 +156,16 @@ class ProcessGroupNCCL : public ProcessGroup {
   // The CUDA events used to sync NCCL streams
   std::unordered_map<std::string, std::vector<CUDAEvent>> ncclEvents_;
 
-  // Caches the number of GPUs available in the current system
-  int numGPUs_;
-
   // Store copy of pointer to THCState retrieved from ::at::globalContext().
   THCState* thcState_;
+
+  // ID of this process group
+  std::string processGroupId_;
```

Comment:
Can you name the var "ID", so we have consistency of abbreviations or common names (e.g. ID, NCCL, CUDA, GPU).